controller/packageApi.go

```diff
@@ -64,12 +66,14 @@ func (a *API) PackageApiCreate(w http.ResponseWriter, r *http.Request) {
 
 	// Ensure size limits
 	if len(f.Spec.Source.Literal) > int(fission.ArchiveLiteralSizeLimit) {
-		err := fission.MakeError(fission.ErrorInvalidArgument, "Package literal larger than 256K")
+		err := fission.MakeError(fission.ErrorInvalidArgument,
+			fmt.Sprintf("Package literal larger than 0x%s", strconv.FormatInt(fission.ArchiveLiteralSizeLimit, 16)))
```

Comment:
The error is for humans, plain old decimal is probably better.  Also, please include units (bytes/kb/etc).